src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -62,11 +62,13 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit id: String): Try[Unit] = {
```

Comment:
In `getBagFromDir` the implicit parameter was there for nothing.  After I removed it, I could also remove it from three other methods.